apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/index.js

```diff
@@ -228,14 +226,15 @@ class PageTemplateModal extends Component {
 		// Make sure all blocks use local assets before inserting.
 		this.maybePrefetchAssets( blocks )
 			.then( blocksWithAssets => {
+				this.setState( { isLoading: false } );
```

Comment:
Curious why this bit needed to be added here?

Reply:
When I reopened the modal, sometimes it would still show the "loading" UI indefinitely. So I think there was a case where the old component was being used with the old state, so I'm being extra careful here to set loading to false when the loading bit finishes :)